backend/fms_core/templates.py

```diff
@@ -97,6 +97,29 @@
   "prefill info": [],
 }
 
+LIBRARY_CONVERSION_TEMPLATE = {
+  "identity": {"description": "Template to convert libraries", "file": static("submission_templates/Library_conversion_v3_9_0.xlsx")},
+  "sheets info": [
+      {
+          'name': 'Conversion Batch',
+          'headers': ['Library Batch ID', 'Date (YYYY-MM-DD)', 'Platform', 'Comment',
+                      'Technician Name', 'Kit Used', 'Kit Lot', 'Thermocycler Used', 'PCR Cycles'],
+      },
+      {
+          'name': 'Library',
+          'headers': ['Library Batch ID', 'Library Source Container Barcode', 'Library Source Container Coordinates',
+                      'Destination Library Container Barcode', 'Destination Library Container Coordinates',
+                      'Destination Library Container Name', 'Destination Library Container Kind',
+                      'Destination Library Parent Container Barcode', 'Destination Library Parent Container Coordinates',
+                      'Library Source Concentration (ng/uL)', 'Library Size (bp)', 'Input used for conversion (ng)',
+                      'Volume Used (uL)', 'Volume (uL)', 'Comment'],
+      },
+  ],
+  "prefill info": [
+      ("Library", "Library Source Container Barcode", "container__barcode"),
+      ("Library", "Library Source Container Coordinates", "coordinates"),],
```

Comment:
We should prefill "Library Source Concentration (ng/uL) and "Library Size (bp)" too.